testsuite/ext-tools/json_creation_script.py

```diff
@@ -0,0 +1,168 @@
+import argparse
+import json
+import subprocess
+import sys
+
+import requests
```

Comment:
It did work on my side with the command suggested on the [request docs](https://pypi.org/project/requests/):

```bash
python3 -m pip install requests
Defaulting to user installation because normal site-packages is not writeable
Collecting requests
  Using cached requests-2.28.1-py3-none-any.whl (62 kB)
Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3.10/site-packages (from requests) (3.3)
Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3.10/site-packages (from requests) (2021.10.8)
Requirement already satisfied: charset-normalizer<3,>=2 in /usr/lib/python3.10/site-packages (from requests) (2.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3.10/site-packages (from requests) (1.26.11)
Installing collected packages: requests
Successfully installed requests-2.28.1
```

Reply:
It did work for me on the latest openSUSE Tumbleweed.

Reply:
@Bischoff have you tried adding a shebang at the beginning of the script? :wink: I had the same problem with the Ruby script from Oscar.